tests/testdata/plugins/test-plugin-with-errors/enqueued-scripts-size-check.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * File contains errors for the enqueued script sizes check.
+ */
+
+function plugin_check_test_enqueue_large_script() {
+	wp_enqueue_script(
+		'plugin_check_large_script',
+		plugin_dir_url( __FILE__ ) . 'assets/large-script.js'
+	);
+}
+
+add_action( 'wp_enqueue_scripts', 'plugin_check_test_enqueue_large_script' );
```

Comment:
This code allows us to test the main script size. We should also find a way to test the logic in `Enqueued_Scripts_Size_Check` that covers inline scripts.

We could for example add an inline script here, and in the test set the threshold to a value that will only be exceeded by the main script _plus_ the inline script.

Reply:
Thanks @felixarntz I've added an inline script and used the `$threshold_size` we pass in the constructor to test when the inline script goes over the threshold.